cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -110,7 +110,7 @@ func init() {
 
 	describeClusterClusterCmd.Flags().BoolVar(&dc.disableNoEcho, "disable-no-echo", false, ""+
 		"Disable hiding of a MachineInfrastructure and BootstrapConfig when ready condition is true or it has the Status, Severity and Reason of the machine's object.")
-	describeClusterClusterCmd.Flags().BoolVar(&dc.disableGrouping, "disable-grouping", false,
```

Comment:
The old flag should be deprecated, but it can't be removed.
We can then combine them in a way that if grouping is disabled either with disable-grouping=true or with grouping=false

Reply:
have added back that option with a comment

Reply:
The comment is not enough
Please see https://pkg.go.dev/github.com/spf13/pflag#readme-deprecating-a-flag-or-its-shorthand about how to deprecate the flag.
Also, the old flag should be considered in the code (there is another comment above).